cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -134,7 +134,7 @@ public void start(Future<Void> start) {
 
     @Override
     @SuppressFBWarnings(value = "BC_UNCONFIRMED_CAST")
```

Comment:
Do we still needs this annotation?

Reply:
You are absolutely right, this is no longer needed. Removing now.